e2e/fixtures/hearingDetails.js

```diff
@@ -0,0 +1,40 @@
+module.exports = [
+  {
+    caseManagement: 'Case management',
+    venue: 'Venue',
+    date: {
+      day: '01',
+      month: '01',
+      year: '2050',
+    },
+    preHearingAttendance: 'Pre hearing attendance',
+    hearingTime: 'hearing time',
+    hearingType: {
+      interpreter: 'Interpreter',
+      welsh: 'Spoken or written Welsh',
+      somethingElse: 'Something else',
+    },
+    giveDetails: 'Some details',
```

Comment:
Could we rename this to `hearingTypeDetails`

Reply:
Yes:)

Reply:
Renamed.